codespell_lib/data/dictionary.txt

```diff
@@ -16443,6 +16598,7 @@ shortcomming->shortcoming
 shortcommings->shortcomings
 shortend->shortened, short end,
 shortern->shorten
+shortkuts->shortcuts
```

Comment:
i guess we can also add `shortkut->shortcut` while we're at it.

Reply:
Added